src/components/molecules/search-results-list/results-list-item.jsx

```diff
@@ -30,9 +35,24 @@ const ResultsListItem = ({ result = {}, language = 'en' }) => {
 			);
 		}
 	}
+
+	const handleResultItemTitleClick = () => {
+		const pageUnit = urlQuery.get('pageunit');
```

Comment:
You should never be referencing query params except in either the route or the main view. This does not lend itself to easy testing since you have to mock a full url just to test if this component can display.

Reply:
Removed!